README.md

```diff
@@ -118,8 +118,6 @@ These are build-time tools packaged as [Node](http://nodejs.org/) modules.
 
 * [PhantomJS](https://github.com/Medium/phantomjs)&#8201;&mdash;&#8201;NPM wrapper for
   grabbing the right PhantomJS binary.
-* [Plovr](https://github.com/Medium/plovr)&#8201;&mdash;&#8201;Integration of
```

Comment:
oh wait this is the opensource repo, ya, we no integrated our fork with main plovr